docs/consume-packages/Package-References-in-Project-Files.md

```diff
@@ -170,6 +170,20 @@ Conditions can also be applied at the `ItemGroup` level and will apply to all ch
 </ItemGroup>
 ```
 
+When you have many targets, it may be better to match ranges of TFMs, in which case you can use [`IsTargetFrameworkCompatible`](/visualstudio/msbuild/property-functions#msbuild-targetframework-and-targetplatform-functions):
+```xml
+<ItemGroup>
```

Comment:
The 3 examples are pretty equivalent. 

They're all showing the usage of IsTargetFrameworkCompatible with negation. 
I think it'll be easier for the reader if they were to see one example.